### DISCUSSION TITLE
is it available?

### DISCUSSION THREAD

**Discussion by onlining** [oliver-zehentleitner/unicorn-binance-websocket-api#350] [Q&A]
i think using websocket. we couldn't get new order and cancel order. i mean something realted to each user's personal information. 
is it right? or i can order using websocket?. i already test using restapi. but i failed using websocket

**Comment by oliver-zehentleitner** [comment]
Yes of course! 

Create a new order: https://unicorn-binance-websocket-api.docs.lucit.tech/unicorn_binance_websocket_api.html#unicorn_binance_websocket_api.api.BinanceWebSocketApiApi.create_order

Please follow this article: https://medium.lucit.tech/create-and-cancel-orders-via-websocket-on-binance-7f828831404
